src/Components/user.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import users from './api/users';
+
+const User = () => {
+  return users.map(user => user.name)
+};

```

Comment:
тут тоже в пропс должен приходить объект user, который ты достаешь из todo в компоненте Todo

Reply:
```suggestion
const User = ({ user }) => (
  <p>{user.name}</p>
);

```
Компонент должен возвращать верстку, заворачивай данные в теги